Tests/ApolloTests/MultipartFormDataTests.swift

```diff
@@ -0,0 +1,133 @@
+//
+//  MultipartFormDataTests.swift
+//  Apollo
+//
+//  Created by Ellen Shapiro on 9/21/20.
+//  Copyright © 2020 Apollo GraphQL. All rights reserved.
+//
+
+import XCTest
+import Apollo
+
+class MultipartFormDataTests: XCTestCase {
```

Comment:
Moved this out of the request creator tests to disambiguate what's actually being tested here